xilem_core/src/view.rs

```diff
@@ -113,9 +113,9 @@ pub trait View<State, Action, Context: ViewPathTracker, Message = DynMessage>:
 }
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
-/// An identifier used to differentiation between the direct children of a [View].
+/// An identifier used to differentiation between the direct children of a [`View`].
```

Comment:
IIRC, this one was also intentional, but is more marginal. I think with the capitalisation as-is, it's correct to change it.

Reply:
As long as it is capitalized here, I think it should have backticks, so I've not updated my PR.